plugins/catalog-backend/migrations/20221201085245_add_last_updated_at_in_final_entities.js

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
While we're doing that last thing, maybe add a ts-check comment here and the doc comment on `up` too, similar to here https://github.com/backstage/backstage/blob/24b74c35792742327763e2e1513b4df643ee155d/plugins/catalog-backend/migrations/20221109192547_search_add_original_value_column.js#L17

Reply:
Hrm, I know I'm pushing it a bit but for bonus points, if you have time, you could also make a migration test [here](https://github.com/backstage/backstage/blob/24b74c35792742327763e2e1513b4df643ee155d/plugins/catalog-backend/src/migrations.test.ts#L47) to make sure that the migration works well for all database engines (while there's at least one fake entry in the `final_entities` table).

Reply:
thanks for advice! I have added ts-check and migration testing.